src/test/java/lotto/domain/RankBoardTest.java

```diff
@@ -0,0 +1,88 @@
+package lotto.domain;
+
+import lotto.util.IntToLottoConverter;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class RankBoardTest {
+
+    private WinningLotto winningLotto;
+
+    @BeforeEach
+    void setUp() {
+        Lotto winningNumbers = IntToLottoConverter.toLotto(List.of(1,2,3,4,5,6));
```

Comment:
```suggestion
        Lotto winningNumbers = IntToLottoConverter.toLotto(List.of(1, 2, 3, 4, 5, 6));
```

구분자 사이에 공백을 줘서 가독성을 높혀볼까요?